src/model/migrations.ts

```diff
@@ -702,10 +703,28 @@ export default async function runMigrations() {
 
   migrations.push(v18);
 
+  /**
+   *************** Migration v19 ******************
+   Purge walletconnect v2 sessions with no accounts
+   */
+  const v19 = async () => {
```

Comment:
can we add this to the new migrations flow that our king @estrattonbailey  made? wanna deleeet these at some point

Reply:
totally forgot that existed lol

Reply:
am I having a stroke or did I respond to a notification like this months ago lmao

miss you guysssss ❤️

Reply:
@estrattonbailey !!!!!

hope all is well over at bsky

Reply:
ill never miss a chance to tag my king